.github/workflows/tests.yaml

```diff
@@ -97,10 +102,11 @@ jobs:
         run: |
           make test-integrations
       - name: Run overlay smoke test
+        if: ${{ matrix.adjective == 'focal' }} #TODO: fix this smoke test for jammy
```

Comment:
failure if enabled: https://github.com/canonical/craft-parts/actions/runs/3339121000/jobs/5527668931

Reply:
This is interesting, because `/usr/bin/apt-key` is ithere and can be executed. I wonder if this is related to the bad character device problem we had in rockcraft.